src/app/pages/home/home.component.ts

```diff
@@ -1,18 +1,45 @@
-import {Component, OnInit, ViewChild} from '@angular/core';
+import { Component, OnInit, ViewChild, OnDestroy } from '@angular/core';
+import { ToastrService } from 'ngx-toastr';
+
+import { PropertiesService } from 'src/app/services/properties/properties.service';
+import { Property } from 'src/app/models/Property';
+
+import { environment } from 'src/environments/environment';
+import { Subscription } from 'rxjs';
 
 @Component({
   selector: 'app-home',
   templateUrl: './home.component.html',
   styleUrls: ['./home.component.scss']
 })
-export class HomeComponent implements OnInit {
+export class HomeComponent implements OnInit, OnDestroy {
   // @ts-ignore
   @ViewChild('searchForm') form: any;
+  subscription: Subscription;
+  trendingPropertiesUrl = `${environment.api_url}/properties/trending?city=`;
+  trendingProperties: Property[] = [];
 
-  constructor() {
-  }
+  constructor(
+    private propertiesService: PropertiesService,
+    private toastService: ToastrService
+  ) {}
 
   ngOnInit() {
+    this.subscription = this.propertiesService
+      .getProperties(this.trendingPropertiesUrl)
+      .subscribe(
+        ({ data: { property } }) => {
+          this.trendingProperties = property.slice(0, 6);
+        },
+        _ => {
```

Comment:
Great work @Tevinthuku, for readability could we name this `_` to something more intuitive like `error`

Reply:
Hey @CryceTruly , Im not using the `errror object` coming from the api. The underscore is actually valid typescript syntax. The message Ive setup is more user friendly in case anything happens and the properties couldnt be fetched.

https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-0.html#flag-unused-declarations-with---nounusedparameters-and---nounusedlocals

Reply:
👍